src/main/kotlin/org/rust/ide/formatter/RustSpacing.kt

```diff
@@ -0,0 +1,118 @@
+package org.rust.ide.formatter
+
+import com.intellij.formatting.SpacingBuilder
+import com.intellij.psi.codeStyle.CommonCodeStyleSettings
+import com.intellij.psi.tree.IElementType
+import com.intellij.psi.tree.TokenSet
+import org.rust.lang.core.psi.RustCompositeElementTypes.*
+import org.rust.lang.core.psi.RustKeywordTokenType
+import org.rust.lang.core.psi.RustTokenElementTypes.*
+import com.intellij.psi.tree.TokenSet.create as ts
```

Comment:
I meant to use `tokenSetOf` not only in this file, but else where (`RustElementTypes`, for example).  Doing rename in every file seems clumsy. But this does not really matter.